packages/fxa-auth-server/test/local/routes/subscriptions/stripe-webhooks.js

```diff
@@ -486,6 +517,78 @@ describe('StripeWebhookHandler', () => {
       });
     });
 
+    describe('handleProductUpdatedEvent', () => {
+      let scopeContextSpy, scopeSpy;
+      beforeEach(() => {
+        scopeContextSpy = sinon.fake();
+        scopeSpy = {
+          setContext: scopeContextSpy,
+        };
+        sandbox.replace(Sentry, 'withScope', (fn) => fn(scopeSpy));
+      });
+
+      it('throws a sentry error if the update event data is invalid', async () => {
+        const updatedEvent = deepCopy(eventProductUpdated);
+        await StripeWebhookHandlerInstance.handleProductUpdatedEvent(
+          {},
+          updatedEvent
+        );
+
+        assert.isTrue(scopeContextSpy.called, 'Expected to call Sentry');
```

Comment:
Would be nice if we verify that `fetchPlansByProductId` was called.  Also would be nice if we know that number of Sentry calls equals to the number of mocked plans returned by `fetchPlansByProductId` that will fail validation.  I don't remember seeing other tests verifying the `scope.setContext` and `Sentry.captureMessage` calls; this seems like a good place for that.